xml/System.IO/StreamReader.xml

```diff
@@ -553,10 +553,24 @@
         <Parameter Name="options" Type="System.IO.FileStreamOptions" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="path">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="path">The complete file path to be read.</param>
+        <param name="options">An object that specifies the configuration options for the underlying <see cref="T:System.IO.FileStream"/>.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.IO.StreamReader" /> class for the specified file path, using the default encoding, enabling detection of byte order marks at the beginning of the file, and configured with the specified <see cref="T:System.IO.FileStreamOptions"/> object.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException"><paramref name="stream" /> is not readable.
+
+-or-
+
+          <paramref name="path" /> is an empty string ("").</exception>
```

Comment:
nit: extra empty lines

```suggestion
-or-
          <paramref name="path" /> is an empty string ("").</exception>
```

Reply:
Those are needed here to ensure they render properly in html. It's in triple slash where we can remove them to save code lines.

Reply:
@carlossanlop thanks, I was not aware of that!